helm/values.schema.json

```diff
@@ -0,0 +1,1710 @@
+            },
+            "additionalProperties": true
+          },
+          "nodeSelector": {
+            "$id": "#/properties/deployment/properties/nodeSelector",
+            "type": "object",
+            "title": "The nodeSelector schema",
+            "description": "Node Selector for Connaisseur pod. (https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector)",
+            "examples": [
+              {
+                "disktype": "ssd"
+              }
+            ],
+            "additionalProperties": true
+          },
+          "tolerations": {
+            "$id": "#/properties/deployment/properties/tolerations",
+            "type": "array",
+            "title": "The tolerations schema",
+            "description": "Tolerations for Connaisseur pod. (https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)",
+            "examples": [
+              [
+                {
+                  "key": "app.kubernetes.io/instance",
+                  "operator": "Equal",
+                  "value": "no-connaisseur",
+                  "effect": "NoSchedule"
+                }
+              ]
+            ],
+            "additionalItems": true,
+            "items": {
+              "$id": "#/properties/deployment/properties/tolerations/items"
+            }
+          },
+          "affinity": {
+            "$id": "#/properties/deployment/properties/affinity",
+            "type": "object",
+            "title": "The affinity schema",
+            "description": "Affinity for Connaisseur pod.",
+            "examples": [
+              {
+                "podAntiAffinity": {
+                  "preferredDuringSchedulingIgnoredDuringExecution": [
+                    {
+                      "podAffinityTerm": {
+                        "labelSelector": {
+                          "matchExpressions": [
+                            {
+                              "key": "app.kubernetes.io/instance",
+                              "operator": "In",
+                              "values": [
+                                "connaisseur"
+                              ]
+                            }
+                          ]
+                        },
+                        "topologyKey": "kubernetes.io/hostname"
+                      },
+                      "weight": 100
+                    }
+                  ]
+                }
+              }
+            ],
+            "additionalItems": true,
+            "items": {
+              "$id": "#/properties/deployment/properties/tolerations/items"
+            }
+          },
+          "securityContext": {
+            "$id": "#/properties/deployment/properties/securityContext",
+            "type": "object",
+            "title": "The securityContext schema",
+            "description": "Security Context of Connaisseur container. (https://kubernetes.io/docs/tasks/configure-pod-container/security-context/)",
```

Comment:
If security context is an Eigenname, it should probably be SecurityContext, otherwise probably lower case c